### PR TITLE
feat: added header block

### PR DESCRIPTION
**Pull request by sergiykostiv**

### CODE REVIEW

File: css/fonts.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@font-face {
+  font-family: "NunitoSans";
+  font-weight: 700;
+  src: url("../fonts/NunitoSans-Bold.woff2") format("woff2"), url("../fonts/NunitoSans-Bold.woff") format("woff");
+}
```
Comment (PetrKulko) [inline css/fonts.css:5]: Do I need a space between the structures here?

File: index.html [hunk 2]
```diff
@@ -1,10 +1,44 @@
...
+                    <a class="header__btn header__register" href="#">Register</a>
+                </div>
+            </div>
+        </header>
+        <main class="main"></main>
+        <footer class="footer"></footer>
+    </div>
 </body>
-</html>
+</html>
```
Comment (PetrKulko) [inline index.html:44]: need empty line

File: scss/fonts.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  src: url('../fonts/Quicksand-Bold.woff2') format('woff2'),
+  url('../fonts/Quicksand-Bold.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'Quicksand';
+  font-weight: 400;
+  src: url('../fonts/Quicksand-Regular.woff2') format('woff2'),
+  url('../fonts/Quicksand-Bold.Regular') format('woff');
+}
```
Comment (PetrKulko) [inline scss/fonts.scss:27]: need empty line

File: index.html [hunk 4]
```diff
@@ -1,10 +1,44 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
+    <title>Livid</title>
 </head>
 <body>
-    <h1>Hello world</h1>
+    <div class="wrapper">
+        <header class="header">
+            <div class="container header__wrapper">
+                <a class="header__logo" href="#">
```
Comment (PetrKulko) [inline index.html:15]: Clicking on the logo should provide a redirect to the home page, I suggest adding to `href="/"`

Comment (sergiykostiv) [reply index.html:15]: OK, agreed.

File: scss/fonts.scss (hunk 3 above)
Comment (sergiykostiv) [reply scss/fonts.scss:27]: OK

File: index.html (hunk 2 above)
Comment (sergiykostiv) [reply index.html:45]: OK

File: index.html [hunk 5]
```diff
@@ -1,10 +1,43 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="static/css/style.css">
+    <title>Livid</title> 
 </head>
 <body>
-    <h1>Hello world</h1>
+    <div class="wrapper">
+        <header class="header">
+            <div class="container header__wrapper">
+                <a class="header__logo" href="/"><img class="header__logo-img" src="/static/img/logo.svg" alt="header_logo"></a>
```
Comment (PKulkoRaccoonGang) [inline index.html:15]: I think this `"/"` inner the `href` not needed

File: index.html [hunk 6]
```diff
@@ -1,10 +1,43 @@
...
+                <nav class="menu header__menu">
+                    <div class="menu__burger"><span class="menu__burger-span"></span></div>
```
Comment (PKulkoRaccoonGang) [inline index.html:17]: I propose that you use a `cursor pointer` for this element, which is a clickable element. And also for the close button.

File: index.html [hunk 7]
```diff
@@ -1,10 +1,43 @@
...
+                            <a class="menu__link" href="#">Register</a>
+                            <span></span>
+                            <a class="menu__link" href="#">Login</a>
+                        </li>
+                    </ul>
+                </nav>
+                <div class="header__buttons">
+                    <button class="header__button header__button-login"><a class="header__button-link" href="#">Login</a></button>
+                    <button class="header__button header__button-register"><a class="header__button-link" href="#">Register</a></button>
+                </div>
```
Comment (PKulkoRaccoonGang) [inline index.html:33]: I think duplicate tags it`s not good. Please remove these duplicate tags.

File: index.html [hunk 8]
```diff
@@ -1,10 +1,43 @@
...
+            </div>
+        </header>
+        <main class="main">
+        </main>
```
Comment (PKulkoRaccoonGang) [inline index.html:37]: Move this closed tag `main` up line

File: scss/style.scss [hunk 9]
```diff
@@ -0,0 +1,339 @@
+@import 'fonts.scss';
+@import 'variables.scss';
+
+
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:4]: Remove this empty line

File: scss/style.scss [hunk 10]
```diff
@@ -0,0 +1,339 @@
...
+}
+
+body {
+  position: relative;
+  margin: 0;
+  overflow: auto;
+  font-size: 16px;
+
+  &.active {
+    overflow: hidden;
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:27]: Why do we use `overflow: hidden` for `body`?

File: scss/style.scss [hunk 11]
```diff
@@ -0,0 +1,339 @@
...
+  text-decoration: none;
+  color: none;
+}
+
+.container {
+  max-width: 1160px;
+  margin: 0 auto;
+}
+
+@media (max-width: 1024px){
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:41]: Add space after `(max-width: 1024px)`. For example:
`@media (max-width: 1024px) {`

File: scss/style.scss [hunk 12]
```diff
@@ -0,0 +1,339 @@
...
+  .container {
+      padding: 0 10px;
+  }
+}
+
+.header {
+  position: fixed;
+  width: 100%;
+  font-family: 'Quicksand';
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:50]: We use `font-family: 'Quicksand', sans-serif;` for tag `html`. Do we need this duplicate definition?

File: scss/style.scss [hunk 13]
```diff
@@ -0,0 +1,339 @@
...
+  &__button {
+    text-align: center;
+    min-width: 129px;
+    padding: 10px 0;
+    transition: all .3s;
+    border: none;
+    cursor: pointer;
+
+    &:active {
+      box-shadow: 0 0px 0 transparent;
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:95]: Please remove px. For example: 0px => 0

File: scss/style.scss [hunk 14]
```diff
@@ -0,0 +1,339 @@
...
+      transform: translateY(-2px);
+    }
+
+    &-link {
+      color: #FFF;
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:100]: We have `$white` SCSS variables for the `#fff` color.

File: scss/style.scss [hunk 15]
```diff
@@ -0,0 +1,339 @@
...
+      border-radius: 100px;
+      box-shadow: 4px 14px 34px rgba(139, 116, 251, 0.4);
+
+      &:hover {
+        box-shadow: 0 5px 0 #917AFD;
+        transform: translateY(-5px);
+      }
+
+      &:active {
+        box-shadow: 0 0px 0 transparent;
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:116]: Please remove px. For example 0px => 0

File: scss/style.scss [hunk 16]
```diff
@@ -0,0 +1,339 @@
...
+    &-link {
+      color: #FFF;
+      font-weight: 700;
+      line-height: 20px;
+    }
+
+    &-login {
+      background: linear-gradient(135.53deg, #917AFD 2.61%, #6246EA 96.58%);
+      border-radius: 100px;
+      box-shadow: 4px 14px 34px rgba(139, 116, 251, 0.4);
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:108]: Please remove 0 in the alpha channel. For example 0.4 => .4

File: scss/style.scss [hunk 17]
```diff
@@ -0,0 +1,339 @@
...
+      border-radius: 100px;
+      box-shadow: 4px 14px 19px rgba(255, 156, 41, 0.33);
+
+      &:hover {
+        box-shadow: 0 5px 0 #FFB258;
+        transform: translateY(-5px);
+      }
+
+      &:active {
+        box-shadow: 0 0px 0 transparent;
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:132]: Please remove px. For example 0px => 0

File: scss/style.scss [hunk 18]
```diff
@@ -0,0 +1,339 @@
...
+        transform: translateY(-2px);
+      }
+    }
+  
+    &-register {
+      background: linear-gradient(299.27deg, #FF9C29 24.69%, #FFB258 81.84%);
+      border-radius: 100px;
+      box-shadow: 4px 14px 19px rgba(255, 156, 41, 0.33);
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:124]: Please remove 0 in the alpha channel. For example 0.33 => .33

File: scss/style.scss [hunk 19]
```diff
@@ -0,0 +1,339 @@
...
+
+    // &__btn {
+    //   padding: 0;
+    //   background: transparent;
+    //   box-shadow: none;
+      
+      
+    //   &-link {
+    //     color:#6246EA;
+    //   }
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:181]: Do we need these comments?

File: scss/style.scss [hunk 20]
```diff
@@ -0,0 +1,339 @@
...
+      transform: translateY(-10px);
+    }
+
+    &::before {
+      transform: translateY(10px);
+    }
+  }
+
+    &-active {
+
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:231]: Remove this empty line

File: scss/style.scss [hunk 21]
```diff
@@ -0,0 +1,339 @@
...
+  }
+
+  &__list {
+    display: flex;
+    align-items: center;
+    column-gap: 60px;
+  }
+
+  &__item {
+
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:253]: Remove this empty line

File: scss/style.scss [hunk 22]
```diff
@@ -0,0 +1,339 @@
...
+    position: relative;
+    color: $accent-color;
+    padding-bottom: 4px;
+
+    &:hover {
+      color: lighten($accent-color, 10%);
+    }
+
+    &-active {
+
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:269]: Remove this empty line

File: scss/style.scss [hunk 23]
```diff
@@ -0,0 +1,339 @@
...
+        height: 2px;
+        background: $accent-color;
+      }
+    }
+  }
+}
+
+@media (max-width: 1024px){
+  .menu {
+
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:285]: Remove this empty line

File: scss/style.scss [hunk 24]
```diff
@@ -0,0 +1,339 @@
...
+    &__list {
+      column-gap: 30px;
+    }
+  }
+}
+
+@media (max-width: 768px){
+
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:293]: Remove this empty line

File: index.html [hunk 25]
```diff
@@ -1,10 +1,43 @@
...
+        <footer class="footer"></footer>
+    </div>
+
+    <script src="/js/main.js"></script>
```
[the remark this answers is not in the document]
Comment (sergiykostiv) [reply index.html:41]: ok, I fix it in the next commit

File: index.html (hunk 6 above)
Comment (sergiykostiv) [reply index.html:17]: ok, I fix it in the next commit

File: index.html (hunk 7 above)
Comment (sergiykostiv) [reply index.html:33]: ok

File: index.html (hunk 8 above)
Comment (sergiykostiv) [reply index.html:37]: ok

File: index.html (hunk 5 above)
[the remark this answers is not in the document]
Comment (sergiykostiv) [reply index.html:15]: I added '/' because of the previously made comment, remove this?

File: scss/style.scss (hunk 9 above)
Comment (sergiykostiv) [reply scss/style.scss:4]: ok

File: scss/style.scss (hunk 10 above)
Comment (sergiykostiv) [reply scss/style.scss:27]: If we don't use overflow: hidden for the body element, when the menu is open on mobile versions and the user scrolls, the content under the open menu will also scroll, which I consider not desirable.

File: scss/style.scss (hunk 11 above)
Comment (sergiykostiv) [reply scss/style.scss:41]: ok

File: scss/style.scss (hunk 13 above)
Comment (sergiykostiv) [reply scss/style.scss:95]: ок

File: scss/style.scss (hunk 14 above)
Comment (sergiykostiv) [reply scss/style.scss:100]: ok

File: scss/style.scss (hunk 15 above)
Comment (sergiykostiv) [reply scss/style.scss:116]: ok

File: scss/style.scss (hunk 18 above)
Comment (sergiykostiv) [reply scss/style.scss:124]: ok

File: scss/style.scss (hunk 24 above)
Comment (sergiykostiv) [reply scss/style.scss:293]: ok

File: scss/style.scss (hunk 23 above)
Comment (sergiykostiv) [reply scss/style.scss:285]: ok

File: scss/style.scss (hunk 22 above)
Comment (sergiykostiv) [reply scss/style.scss:269]: ok

File: scss/style.scss (hunk 21 above)
Comment (sergiykostiv) [reply scss/style.scss:253]: ok

File: scss/style.scss (hunk 20 above)
Comment (sergiykostiv) [reply scss/style.scss:231]: ok

File: scss/style.scss (hunk 19 above)
Comment (sergiykostiv) [reply scss/style.scss:181]: I think not

File: scss/style.scss [hunk 26]
```diff
@@ -0,0 +1,216 @@
...
+    border: none;
+    cursor: pointer;
+
+    &:active {
+      box-shadow: 0 0px 0 transparent;
+      transform: translateY(-2px);
+    }
+
+    &-link {
+      color: #fff;
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:87]: $white

File: scss/style.scss [hunk 27]
```diff
@@ -0,0 +1,216 @@
...
+      }
+
+      &::before {
+        transform: translateY(10px);
+      }
+    }
+
+    &-active {
+      span {
+        background: transparent;
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:168]: `::after {
        transform: rotate(-45deg);
}

::before {
        transform: rotate(45deg);
}`

File: scss/style.scss [hunk 28]
```diff
@@ -0,0 +1,216 @@
...
+  min-height: 100vh;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.header {
+  position: fixed;
+  z-index: 5;
+  width: 100%;
+  font-family: "Quicksand";
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:48]: Do we need this property here? We asked for the tag `html`.

File: scss/_media.scss [hunk 29]
```diff
@@ -0,0 +1,103 @@
...
+    }
+
+    &__button {
+      min-width: 100px;
+      padding: 8px 0;
+    }
+  }
+
+  .menu {
+    &__list {
```
Comment (PKulkoRaccoonGang) [inline scss/_media.scss:21]: .menu .header__list { ... }

File: scss/_media.scss [hunk 30]
```diff
@@ -0,0 +1,103 @@
...
+        flex-direction: column;
+        width: 100%;
+      }
+
+      &__link {
+        display: block;
+        width: 100%;
+
+        &-active {
+          &::before {
```
Comment (PKulkoRaccoonGang) [inline scss/_media.scss:69]: &-active::before { ... }

File: scss/style.scss [hunk 31]
```diff
@@ -0,0 +1,216 @@
...
+    position: relative;
+    color: $accent-color;
+    padding-bottom: 4px;
+
+    &:hover {
+      color: lighten($accent-color, 10%);
+    }
+
+    &-active {
+      &::before {
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:203]: Please 1 line

File: scss/style.scss [hunk 32]
```diff
@@ -0,0 +1,216 @@
...
+  }
+
+  &__list {
+    display: flex;
+    align-items: center;
+    column-gap: 60px;
+  }
+
+  &__item {
+    &-login {
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:188]: Please 1 line

File: index.html [hunk 33]
```diff
@@ -2,9 +2,39 @@
...
+    <link rel="stylesheet" href="static/css/style.css">
+    <title>Livid</title> 
 </head>
 <body>
-    <h1>Hello world</h1>
+    <div class="wrapper">
+        <header class="header">
+            <div class="container">
+                <div class="header__wrapper">
+                    <a class="header__logo" href="/"><img class="header__logo-img" src="static/img/logo.svg" alt="header_logo"></a>
```
Comment (PKulkoRaccoonGang) [inline index.html:15]: Need more detain alt description. For example ("Livid logotype")

File: index.html [hunk 34]
```diff
@@ -2,9 +2,39 @@
...
+                    <nav class="menu header__menu">
+                        <div class="menu__burger"><span class="menu__burger-span"></span></div>
+                        <ul class="menu__list">
+                            <li class="menu__item"><a class="menu__link menu__link-active" href="#">Home</a></li>
```
Comment (PKulkoRaccoonGang) [inline index.html:19]: I think good simple `active` class, but it`s not requirment

File: scss/style.scss (hunk 26 above)
Comment (sergiykostiv) [reply scss/style.scss:87]: yes

File: scss/style.scss (hunk 27 above)
Comment (sergiykostiv) [reply scss/style.scss:168]: ok

File: scss/style.scss (hunk 28 above)
Comment (sergiykostiv) [reply scss/style.scss:48]: not realy. I remove this

File: scss/style.scss [hunk 35]
```diff
@@ -0,0 +1,210 @@
...
+        transform: translateY(-10px);
+      }
+
+      &::before {
+        transform: translateY(10px);
+      }
+    }
+
+    &-active {
+      span {
```
Comment (PKulkoRaccoonGang) [inline scss/style.scss:164]: &-active span {...}

**Pull request merged**